src/app/pages/communication/communication.page.ts

```diff
@@ -216,10 +217,12 @@ export class CommunicationPage extends PracticeableBasePageComponent implements
   }
 
   dispatchCandidateChoseNewEmail(communicationEmail: string): void {
-    this.setCommunicationType(CommunicationPage.email, CommunicationPage.updatedEmail);
-    this.store$.dispatch(communicationPreferencesActions.CandidateChoseEmailAsCommunicationPreference(
-      communicationEmail, CommunicationPage.email,
-    ));
+    if(this.isNewEmailSelected()) {
```

Comment:
being pedantic here but how did this line pass linting checks?

Reply:
ive not idea why it didnt pick it up...will take a look

Reply:
resolved